.gitignore

```diff
@@ -1,5 +1,6 @@
-node_modules/
-lib/
+*.log
```

Comment:
@neoziro just added this line, but I also sorted the lines.

Reply:
Can you please revert the changes, it is a global `.gitignore` stuff.